### PR TITLE
Message css

### PR DESCRIPTION
**Pull request by stg101**
- Add style for messages
- Add color on channel click

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -170,34 +166,50 @@ function findChannelByName(channelName) {
...
   channel.messages.forEach(channelMessage => {
-    const divMessageGroup = document.createElement("hr");
+    const divMessageGroup = document.createElement("div");
     divMessageGroup.className = "message-group";
     let savedTime = new Date(channelMessage.timestamp);
 
     // Comparison of dates for group
     if (savedTime.toDateString() === new Date().toDateString()) {
-      divMessageGroup.innerHTML = "Today";
+      divMessageGroup.innerHTML = "<div>Today</div><hr>";
```
Comment (AngieCristina) [inline public/js/main.js]: Nice  work

Comment (stg101) [reply public/js/main.js]: Thanks

Comment (AngieCristina) [reply public/js/main.js]: Line of code # 191 that console.log Is necessary?

File: public/js/main.js [hunk 2]
```diff
@@ -170,34 +166,50 @@ function findChannelByName(channelName) {
...
+function colorChannel(channelElement) {
+  let channels = document.getElementsByClassName("channel-item");
+  channels = Array.from(channels);
+  channels.forEach(channel => {
+    channel.style.backgroundColor = "";
+    channel.style.color = "rgb(167, 163, 168)";
+  });
+  if (channelElement) {
+    channelElement.style.backgroundColor = "rgb(16,100,163)";
+    channelElement.style.color = "rgb(255,255,255)";
```
Comment (MariethPerez) [inline public/js/main.js]: Good job!!! It is interesting apply styles in css.

**Pull request merged**